llm/llama-3/gui.yaml

```diff
@@ -0,0 +1,42 @@
+# Starts a GUI server that connects to the Llama-3 OpenAI API server.
+# This works with the endpoint.yaml, please refer to llm/llama-3/README.md
+# for more details.
+# Usage:
+#  1. If you have a endpoint started on a cluster (sky launch):
+#     `sky launch -c llama3-gui ./gui.yaml --env ENDPOINT=$(sky status --ip llama3):8000`
+#  2. If you have a SkyPilot Service started (sky serve up) called llama3:
+#     `sky launch -c llama3-gui ./gui.yaml --env ENDPOINT=$(sky serve status --endpoint llama3)`
+# After the GUI server is started, you will see a gradio link in the output and
+# you can click on it to open the GUI.
```

Comment:
Styleguide: We need spaces between paras for readability (otherwise, the current format looks "joined" and is harder to read).
```suggestion
# Starts a GUI server that connects to the Llama-3 OpenAI API server.
#
# This works with the endpoint.yaml, please refer to llm/llama-3/README.md
# for more details.
#
# Usage:
#
#  1. If you have a endpoint started on a cluster (sky launch):
#     `sky launch -c llama3-gui ./gui.yaml --env ENDPOINT=$(sky status --ip llama3):8000`
#  2. If you have a SkyPilot Service started (sky serve up) called llama3:
#     `sky launch -c llama3-gui ./gui.yaml --env ENDPOINT=$(sky serve status --endpoint llama3)`
#
# After the GUI server is started, you will see a gradio link in the output and
# you can click on it to open the GUI.
```

Reply:
Good point! Updated. Thanks!